pkg/pgmodel/migrations/sql/idempotent/base.sql

```diff
@@ -1429,9 +1483,30 @@ CREATE OR REPLACE VIEW SCHEMA_INFO.label AS
     lk.value_column_name,
     lk.id_column_name,
     ARRAY(SELECT value FROM SCHEMA_CATALOG.label l WHERE l.key = lk.key ORDER BY value)
-      AS values
-  FROM SCHEMA_CATALOG.label_key lk;
+      AS values,
+    cardinality(ARRAY(SELECT value FROM SCHEMA_CATALOG.label l WHERE l.key = lk.key))
+  FROM SCHEMA_CATALOG.label_key lk ORDER BY cardinality DESC;
```

Comment:
hmm the values and cardinality columns use the same array, but now uses two scans. For efficiency we should do something like,
```
...
   va.vals as values,
   cardinality(va.values) as num_values
 FROM SCHEMA_CATALOG.label_key lk 
 INNER JOIN LATERAL(SELECT key, array_agg(value ORDER BY value) as vals FROM SCHEMA_CATALOG.label GROUP BY key) 
        AS va ON (va.key = lk.key) 
```